dali/python/nvidia/dali/ops.py

```diff
@@ -21,7 +21,7 @@
 from nvidia.dali.types import _type_name_convert_to_string, _type_convert_value, DALIDataType
 from future.utils import with_metaclass
 
-_blacklisted_ops = set(["MakeContiguous"])
+_blacklisted_ops = set(["MakeContiguous", "nvJPEGDecoderCPUStage", "nvJPEGDecoderGPUStage"])
```

Comment:
there are more than that. I think it's better to wait for @Kh4L implementation that will be more generic and remove all this blacklist